.travis.yml

```diff
@@ -2,10 +2,54 @@ language: ruby
 rvm:
   - 2.6.3
 
-install:
-  - bundle install
-  - npm install eslint --save-dev
-  - ./node_modules/.bin/eslint src/main/content/_assets/js/*.js
+stages:
+  - name: Linting Test
+    if: type IN (push, pull_request, api)
+  - name: Fork Deploy
+    if: (type IN (api)) AND (repo != kabanero-io/kabanero-website)
+  - name: Production Deploy
+    if: (type IN (api)) AND (repo == kabanero-io/kabanero-website)
 
-script:
-  - ./scripts/build_jekyll_maven.sh
\ No newline at end of file
+jobs:
+  include:
+    - stage: "Linting Test"
```

Comment:
since you run build jekyll maven too in this step it would be more than just "Linting Test" maybe split it into 2 stages one linting stage and one jekyll build stage